assets/js/add-task.js

```diff
@@ -0,0 +1,78 @@
+window.addEventListener("load", () => {
+  const form = document.querySelector("#create-task");
+  const inputTask = document.querySelector("#input-task");
```

Comment:
```suggestion
  const inputValue = document.querySelector("#input-value");
```
It is always a best practice to name your variable so that others can understand them easily, so we may pay some extra attention regarding readability while naming.